docs/src/main/asciidoc/amazon-lambda.adoc

```diff
@@ -240,33 +294,469 @@ of things you need to do.  Take a look at the generated example project to get a
+            <outputDirectory>${file.separator}ssl</outputDirectory>
+            <destName>cacerts</destName>
+            <fileMode>644</fileMode>
+        </file>
+        <file>
+            <source>${project.basedir}${file.separator}src${file.separator}main${file.separator}resources${file.separator}bootstrap</source>
+            <outputDirectory>${file.separator}</outputDirectory>
+            <destName>bootstrap</destName>
+            <fileMode>755</fileMode>
+        </file>
+        <file>
+            <source>${project.basedir}${file.separator}src${file.separator}main${file.separator}resources${file.separator}libsunec.so</source>
+            <outputDirectory>${file.separator}ssl</outputDirectory>
+            <destName>libsunec.so</destName>
+            <fileMode>755</fileMode>
+        </file>
+    </files>
+</assembly>
+----
+
+Maven profile to execute the new function.zip:
+
+[source, xml]
+----
+        <profile>
+            <id>zip</id>
+            <activation>
+                <property>
+                    <name>zip</name>
+                </property>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-assembly-plugin</artifactId>
+                        <version>3.1.0</version>
+                        <executions>
+                            <execution>
+                                <id>zip-assembly</id>
+                                <phase>package</phase>
+                                <goals>
+                                    <goal>single</goal>
+                                </goals>
+                                <configuration>
+                                    <finalName>function</finalName>
+                                    <descriptors>
+                                        <descriptor>src/assembly/function-zip.xml</descriptor>
+                                    </descriptors>
+                                    <attach>false</attach>
+                                    <appendAssemblyId>false</appendAssemblyId>
+                                </configuration>
+                            </execution>
+                        </executions>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+----
+
+[[alexa]]
+== Writing Skills with the Amazon Alexa ASK SDK v2
+
+The starting point for an Alexa Skill on AWS Lambda is typically via subclassing `SkillStreamHandler`,
+which implements the RequestStreamHandler::requestHandler.  If you examine the source code for this class it is very simple,
+and can be implemented directly without this helper base class.
+
+NOTE: Quarkus requires a direct implementation of the RequestStreamHandler, as the build process will scan only directly implemented interfaces,
+inherited interfaces will not be seen.
```

Comment:
Why don't we fix that instead?

Reply:
Yes... part of the reason I added this to the docs was to draw attention to this.  If a core member (or someone who understands what to change in the core libraries) could help make the change that would be appreciated.  I'll raise a separate Feature Request item for it.

Reply:
Followed the instruction on #7955 to check for subclasses of the SkillStreamHandler also.  I have this working (2 lines of code, plus runtime bom addition).

Hence, I'm removing all Alexa references from the document.